app/assets/stylesheets/application.scss

```diff
@@ -10,35 +10,24 @@ $govuk-use-legacy-font: false;
 // for guidance
 @import "govuk_publishing_components/govuk_frontend_support";
 @import "govuk_publishing_components/components/back-link";
-@import "govuk_publishing_components/components/breadcrumbs";
-@import "govuk_publishing_components/components/button";
```

Comment:
Button can also be removed from the print stylesheet as it's already included in static.